source/default/_patterns/02-molecules/tabs/_tabs.twig

```diff
@@ -0,0 +1,56 @@
+{#
+/**
+ * @file
+ * tabs
+ *
+ * Available variables for tabs:
+ *  - tabs.label [string]: Used for aria-label for tabslist.
+ *  - tabs.show_title_in_panel [boolean]: Shows title in panel.
+ *  - tabs.panel_title_element [string]: h2|(h3)|span Element to use for panel title.
+ *  - tabs.data [array]: array of tab objects.
+ *    - title [string]: Used for Tab title and Header inside of tab panel.
+ *    - panel [string]: Panel content.
+ */
+#}
+{% set seed = random() %}
+{% set title_element = tabs.panel_title_element|default('h3') %}
+<div class="tabs mb-4">
+  <ul class="tabs__list px-1" role="tablist" {{ tabs.label ? "aria-label='#{tabs.label}'" }}>
```

Comment:
Since `tabs.label` is required, no need for an optional check. Just print `"aria-label={{ tabs.label }}"`.

Reply:
twig doesn't allow us to specify required variables so shouldn't this be a check